### DISCUSSION TITLE
Why has my pipeline been in the Queued state for 12 hours without starting?

### DISCUSSION THREAD

**Discussion by xcatliu** [community/community#57376] [Actions] [answered]
### Select Topic Area

Bug

### Body

Why has my pipeline been in the Queued state for 12 hours without starting?

https://github.com/xcatliu/typescript-tutorial/actions/runs/5186899920/jobs/9351138965
<img width="1105" alt="image" src="https://github.com/community/community/assets/5453359/b6622368-397f-4f52-af3c-7b0bc065c66f">

**Comment by airtower-luna** [comment, marked as answer]
The problem is that your job is trying to use a `ubuntu-18.04` runner. That runner version has been deprecated since last summer, and was finally removed in April, see: https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/

You'll need to update, the current Ubuntu runner version is `ubuntu-22.04`.

**Reply by xcatliu** [reply]
Thanks for your reply, changing to `ubuntu-22.04` works for me. Have a nice day!
